sand/extension_interface/search.py

```diff
@@ -1,21 +1,23 @@
 from abc import ABC, abstractmethod
+from typing import Dict
 
 
 class ISearch(ABC):
     """ Search Interface to support searches from multiple
         KG datastores.
     """
+
     @abstractmethod
-    def find_class_by_name(self):
+    def find_class_by_name(self, search_text: str) -> Dict:
```

Comment:
we should use either `dataclass` or `TypedDict` to provide the exact schema of the returned type

Reply:
converted it to dataclass, enhanced typing for the search API.